lib/fluent/plugin/filter_mesosphere_filter.rb

```diff
@@ -177,9 +181,15 @@ def parse_env(env)
     def merge_json_log(record)
       if record.key?('log')
         log = record['log'].strip
+        namespace = record['namespace']
         if log[0].eql?('{') && log[-1].eql?('}')
           begin
-            record = Oj.load(log).merge(record)
+            log_json = Oj.load(log)
```

Comment:
So here is where we need to go through every top level key and append the namespace... 
So for the following 

```json
{
"data": {
  "env": "foo"
  }
}
```
we would need to append the namespace so the resulting json would be... 

```json
{
"my_namespace": {
  {
    "data": {
       "env": "foo"
    }
  }
}
```